src/common/signalk/SKNMEA2000Parser.cpp

```diff
@@ -99,23 +149,155 @@ const SKUpdate& SKNMEA2000Parser::parse128267(const SKSourceInput& input, const
 
     update->setEnvironmentDepthBelowTransducer(depthBelowTransducer);
 
-    if (!N2kIsNA(offset)) {
-      // When offset is negative, it's the distance between transducer and keel
-      if (offset < 0) {
-        update->setEnvironmentDepthTransducerToKeel(offset * -1);
-        update->setEnvironmentDepthBelowKeel(depthBelowTransducer + offset);
-      }
-      else if (offset > 0) {
-        update->setEnvironmentDepthSurfaceToTransducer(offset);
-        update->setEnvironmentDepthBelowSurface(depthBelowTransducer + offset);
+    if (N2kIsNA(offset)) {
+      // When no offset then offset should be Zero
+      offset = 0;
+    }
+    // When offset is negative, it's the distance between transducer and keel
+    if (offset < 0) {
+      update->setEnvironmentDepthTransducerToKeel(offset * -1);
+      update->setEnvironmentDepthBelowKeel(depthBelowTransducer + offset);
+    }
+    else if (offset > 0) {
+      update->setEnvironmentDepthSurfaceToTransducer(offset);
+      update->setEnvironmentDepthBelowSurface(depthBelowTransducer + offset);
+    }
+
+    _sku = update;
+    return *_sku;
+  }
+  else {
+    DEBUG("Unable to parse N2kMsg with PGN %i", msg.PGN);
+    return _invalidSku;
+  }
+}
+
+// *****************************************************************************
+//   PGN 130306 W I N D
+//    tN2kWindReference:
+//      N2kWind_True_North=0,
+//      N2kWind_Magnetic=1,
+//      N2kWind_Apprent=2,
```

Comment:
Looks like a typo in NMEA2000. @ttlappalainen is this supposed to be `Apprent`? I think `apparent` would be better.